sdk/src/main/java/io/dapr/config/Properties.java

```diff
@@ -45,6 +45,8 @@ public class Properties {
 
   /**
    * Dapr's default use of gRPC or HTTP for Dapr's method invocation APIs.
+   *
+   * @deprecated HTTP support is being deprecated; use gRPC instead. This will be removed in a future release.

```

Comment:
@johnewart had a discussion with @artursouza .... Let's not have this deprecated as yet. Only for service invocation... We need to decide on the route of that

Reply:
Yes, we still need this, unless we provide a new API that is called "invokeHTTP" or something. Then this one can be deprecated.

Reply:
Sure, but (for my understanding) is it common for one to use gRPC to talk to Dapr but not want to use it for method invocation?

Reply:
When the service is gRPC, yes. When the service is HTTP, we want clients to use HTTP invocation and we have received feedback that users like to have a convenient method for invoking HTTP services. So, we need to keep an HTTP specific method in the client.

Reply:
Removed the deprecation notice from this portion of the code.